assets/questions/reverseWords.js

```diff
@@ -20,7 +20,11 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
   title: 'Reverse Words',
   starterCode: {
     python:
-`def reverseWords(s):
+`# Python version 2.7
```

Comment:
@rabidbit -- I missed your comment about this in the issue thread, but one concern I have with this is that it would need to be manually added for every question. This seems like it would be tedious.

Is there a way to put it in the UI instead? For example, could it exist as text above/below the coding window (that might be later replaced by a dropdown when we support multiple languages)?

Reply:
Are you talking about stating what language is being used? If so, my last comment in #572 was to have the language name next to "New to Python?", not be included as a comment (see my screenshot and subsequent discussion on font size). What's included as a comment are question-specific info, like what type the input is. IIRC, further offline team discussion was to use Angular to hide the text language specifying the language and display the drop down based on the number of languages available.